config/routes.rb

```diff
@@ -54,6 +54,8 @@
   namespace :merchant do
     get '/', to: 'dashboard#index'
     get '/items', to: 'items#index'
+    patch '/items/:id/deactivate', to: 'items#update'
+    patch '/items/:id/activate', to: 'items#update'
```

Comment:
We could make this even more efficient by combining into one path:

patch '/items/:id/activate_deactivate', to: 'items#update'